src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -545,43 +545,39 @@ private boolean offerNotEnoughResources(Offer offer, double cpus, int mem) {
   }
 
   private void reconcileTasks(SchedulerDriver driver) {
-    updateReconciliationDeadline();
```

Comment:
If you remove this, shouldn't you at least set "reconciliationCompleted=false"? If the master fails over multiple times in the life of the scheduler, you'll need to clear the flag before starting reconciliation again.